app/routes/public/exhibition/index.js

```diff
@@ -20,6 +30,19 @@ export default class ExhibitionRoute extends Route {
         val  : 'accepted'
       }
     ];
+
+    if (params.search) {
+      filterOptions.push({
+        or: [
```

Comment:
no need to add or there is only one option for search (by name)

Reply:
Name would need to be added to the url search query. I tried without the or, it doesn't work.